src/server/ua_server_internal.h

```diff
@@ -172,6 +173,10 @@ struct UA_Server {
     UA_PubSubManager pubSubManager;
 #endif
 
+#if defined(UA_ENABLE_PUBSUB_ENCRYPTION) && defined(UA_ENABLE_PUBSUB_SKS)
+    LIST_HEAD(PubSubKeyList, UA_PubSubKeyStorage) pubSubKeyList;
+#endif
+
```

Comment:
Could you please move the "pubSubKeyList" into the "pubSubManager"?

Reply:
Yes, I also realized it later that it is better suited in PubSubManager. I did the same thing for SecurityGroups  managed by the PubSub Manager ( not yet Pushed)